resources/tests.mjs

```diff
@@ -857,11 +869,12 @@ Suites.push({
     name: "NewsSite-Next",
     url: "resources/newssite/news-next/dist/index.html#/home",
     tags: ["newssite", "language"],
+    type: "async",
     async prepare(page) {
         await page.waitForElement("#navbar-dropdown-toggle");
     },
     tests: [
-        new BenchmarkTestStep("NavigateToUS", (page) => {
+        new BenchmarkTestStep("NavigateToUS", async (page) => {
```

Comment:
Why is async callback needed here? The function doesn't seem to do anything async, no?

Reply:
removed!